drag_and_drop_v2/translations/en/LC_MESSAGES/text.po

```diff
@@ -286,10 +286,6 @@ msgid ""
 msgstr ""
 
 #: templates/html/js_templates.html
-msgid "Zones"
```

Comment:
I tried it once, but it didn't seem to work. I attributed it to lack of full i18n support in xblocks.